### PR TITLE
Termbox: Paint it

### PR DESCRIPTION
**Pull request by wiese**
Initial styling for termbox when used in `&useformat=mobile`.
This omits layout details, responsiveness to create the playing field for more fine-grained changes. This also omits the development version (`public/index.html`) which could do with _some_ environment for the component to be rendered in (inherit wikibase styles / just use a box / ...) - let's discuss before we invest in either direction.

Please see the individual commits for modular changes and reasoning.

Pin in wikibase needs to be updated after main line merge for changes to come into effect in SSR and CSR ([cf.](https://gerrit.wikimedia.org/r/c/mediawiki/extensions/Wikibase/+/476541)).

Bug: T207469

### CODE REVIEW

File: src/components/App.vue [hunk 1]
```diff
@@ -35,4 +35,7 @@ export default class App extends Vue {
 </script>
 
 <style lang="scss">
+.wikibase-entitytermsview {
+	@import 'reset-css';
```
Comment (bitPogo) [inline src/components/App.vue]: Looks like black magic

File: src/filter/message.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/**
+ * TODO dynamic messages
+ * When state (user language) comes into play, using a computed will likely be
+ * the more correct™ approach - https://stackoverflow.com/a/42829248
+ */
+export default ( key: string ): string => {
+	if ( key === 'wikibase-termbox-alias-separator' ) {
```
Comment (bitPogo) [inline src/filter/message.ts]: Maybe it's good to document this variance

File: src/components/TermBox.vue [hunk 3]
```diff
@@ -1,19 +1,20 @@
 <template>
-	<div class="wikibase-termbox-primaryLanguage">
+	<!-- TODO dynamize wikibase-termbox--primaryLanguage for T205261 -->
+	<div class="wikibase-termbox wikibase-termbox--primaryLanguage">
```
Comment (bitPogo) [inline src/components/TermBox.vue]: 'wikibase-termbox' - depends where the other components are located should be probably should be move one level up.

File: src/components/App.vue (hunk 1 above)
Comment (wiese) [reply src/components/App.vue]: https://meyerweb.com/eric/tools/css/reset/

File: src/components/TermBox.vue (hunk 3 above)
Comment (wiese) [reply src/components/TermBox.vue]: I feel the components should be self-contained and found the **B**EM block class present only with a modifier attached problematic.
Compare "Under the hood" http://getbem.com/introduction/

**Pull request merged**